avail/src/main/kotlin/avail/compiler/ParsingOperation.kt

```diff
@@ -659,53 +663,183 @@ enum class ParsingOperation constructor(
 	},
 
 	/**
-	 * `13` - Reserved for future use.
+	 * `13` - Check whether we are positioned immediately after a newline and
+	 * whitespace that matches the line on which the first token of this phrase
+	 * or its leading argument occurred.  If the indentation string (i.e.,
+	 * spaces and tabs) agrees, continue parsing, otherwise report a weak error
+	 * and abandon this parse theory.
 	 */
-	@Suppress("unused")
-	RESERVED_13(13, false, true)
+	MATCH_INDENT(13, false, true)
 	{
 		override fun execute(
 			compiler: AvailCompiler,
 			stepState: ParsingStepState,
 			instruction: Int,
 			successorTree: A_BundleTree)
 		{
-			throw UnsupportedOperationException(
-				"Illegal reserved parsing operation")
+			skipWhitespaceAndComments(stepState.start) { successorStates ->
+				successorStates.forEach { parsingState ->
+					val state = stepState.copy { start = parsingState }
+					val initialIndent = state.initialIndentationString()
+					val currentIndent = state.currentIndentationString()
+					when
+					{
+						currentIndent === null ->
+						{
+							state.start.expected(STRONG) { accept ->
+								accept(
+									"an indentation to match that at the " +
+										"beginning of the phrase " +
+										"(${initialIndent.q}), but this " +
+										"position is not after leading " +
+										"whitespace on the line")
+							}
+						}
+						!currentIndent.equals(initialIndent) ->
+						{
+							state.start.expected(STRONG) { accept ->
+								accept(
+									"the indentation to exactly match that " +
+										"at the beginning of the phrase. The " +
+										"indentation at the beginning was " +
+										"${initialIndent.q}, but the current" +
```

Comment:
Missing a space after `"current"`, such that `"currentline"` (sic) will be produced.